app/scripts/lib/fxa-client.js

```diff
@@ -112,6 +112,16 @@ define(function (require, exports, module) {
           });
     },
 
+    checkAccountByEmailExists: function (email) {
```

Comment:
I can update function name. I asked @rfk a similar question. Basically, an account is considered `existing` if it has been created. It could have a verified email or unverified email.